src/main/rpm/1-pre-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2020 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#include <service.sh>
+
+NUMBER_OF_INSTALLATIONS=$1
+MODULE_NAME=easy-add-files-to-bag
+PHASE="PRE-INSTALL"
+
+echo "$PHASE: START (Number of current installations: $NUMBER_OF_INSTALLATIONS)"
+service_stop $MODULE_NAME $NUMBER_OF_INSTALLATIONS
+service_create_module_user $MODULE_NAME
```

Comment:
@janvanmansum 
All these RPM files seem service related. Can I indeed drop them all? What about the documentation is telling about RPM?

Reply:
@jo-pol : yes, you can delete the whole `src/main/rpm` folder if this module is command line only. Compare with `easy-manage-deposit`.